### PR TITLE
Update docs: add link to example on how to fill Jira release tickets

### PR DESCRIPTION
**Pull request by ilia-kebets-sonarsource**

### CODE REVIEW

File: docs/RELEASING.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## Releasing npm package
 
-* [Create a Jira release ticket](https://jira.sonarsource.com/projects/REL)
+- [Create a Jira release ticket](https://jira.sonarsource.com/projects/REL) like [this one](https://sonarsource.atlassian.net/browse/REL-2313)
```
Comment (yassin-kammoun-sonarsource) [inline docs/RELEASING.md:3]: Why change from `*` to `-`? It's breaking the Markdown [here](https://github.com/SonarSource/eslint-plugin-sonarjs/blob/9777e71e971141aa6714af3a0daf3ff8bf006732/docs/RELEASING.md).

**Review by yassin-kammoun-sonarsource — CHANGES_REQUESTED** [review]

**Comment by yassin-kammoun-sonarsource** [comment]
@ilia-kebets-sonarsource Could you create another PR that bumps the package version like [this one](https://github.com/SonarSource/eslint-plugin-sonarjs/commit/03b1160df277948fb8d5ff099542050eb03ba93b) ?

File: docs/RELEASING.md (hunk 1 above)
Comment (ilia-kebets-sonarsource) [reply docs/RELEASING.md:3]: I would like to blame my editor for this.

Comment (yassin-kammoun-sonarsource) [reply docs/RELEASING.md]: Let's blame it then ;)

**Review by yassin-kammoun-sonarsource — APPROVED** [review]

**Pull request merged**